catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswQueryFactory.java

```diff
@@ -178,9 +180,11 @@ private CswRecordMapperFilterVisitor buildFilter(QueryConstraintType constraint,
                     null);
         }
         visitor.setVisitedFilter(filter);
-        if (typeNames.contains(new QName(CswConstants.CSW_OUTPUT_SCHEMA,
+        if ((typeNames.contains(new QName(CswConstants.CSW_OUTPUT_SCHEMA,
```

Comment:
You might want to consider breaking this condition into two local `boolean` variables:
```
boolean isCsw (come up with better names for these) = typeNames.contains(new QName(...));
boolean isGmd = typeNames.contains(new QName(...));
if (isCsw || isGmd) {
...
```
The condition is kind of hard to read inline.